pytest/test_io.py

```diff
@@ -44,3 +45,10 @@ def test_xarray_dataset_with_more_data():
     ds = cpt.assemble_dataset(results, mesh=True)
     assert 'nb_faces' in ds.coords
     assert set(ds.coords['nb_faces'].values) == set([b.mesh.nb_faces for b in bodies])
+
+
+def test_dataset_from_bemio():
+    with open('bemio_class.pkl', 'rb') as inp:
+        bemio_data = pickle.load(inp)
```

Comment:
I suppose you used a pickle to avoid explicitly calling the Bemio library. The problem is that it seems to need Bemio to load the pickle.
```
============================== FAILURES ===============================
_______________________ test_dataset_from_bemio _______________________

    def test_dataset_from_bemio():
        with open('bemio_class.pkl', 'rb') as inp:
>           bemio_data = pickle.load(inp)
E           ModuleNotFoundError: No module named 'bemio'

test_io.py:52: ModuleNotFoundError
```
I guess we have no choice but to add Bemio as a (optional?) testing dependency.
If we do so, it might be more robust to load from Bemio data file format rather than from a pickle.

Reply:
I agree. I suppose if a user is using the Bemio importing functionality, it is reasonable to assume that they already have installed Bemio on their local machine. In lieu of this, 3810be5 changes `test_dataset_from_bemio` to use the [`importorskip`](https://docs.pytest.org/en/latest/reference/reference.html#pytest-importorskip-ref) function to import Bemio if it is installed, and skips the test otherwise. Let me know if you think this is a reasonable solution. Note that the test itself still fails as of this commit due to the values being incorrect; see my comment on the overall PR.

As a potentially relevant side note, the Python distribution of Bemio is no longer supported, and is still written in Python 2, so installing the base repo of Bemio would cause issues with this test. I have created a Python 3 compatible fork of Bemio [here](https://github.com/michaelcdevin/bemio/tree/master-python3), which is what I have been using for testing.

Obviously, since Capytaine requires Python 3.6+, any user using Bemio in conjunction will have already addressed the version incompatibility somehow (using my fork or otherwise), so I don't believe a code solution to this is needed (let me know if you think otherwise). However, once I get around to documentation for this feature, I will be sure to mention this point.

Reply:
Unit test is now working as of 2574eaf